app/views/directives/edit-environment-from.html

```diff
@@ -2,20 +2,17 @@
   <div
     ng-if="$ctrl.showHeader"
     class="environment-from-entry environment-from-editor-entry-header">
-    <div class="form-group environment-from-editor-header value-header">
-      <div class="input-group">
+    <div class="environment-from-editor-header config-map-header">
         Config Map/Secret
-      </div>
     </div>
-    <div class="form-group environment-from-editor-header value-header">
-      <div class="input-group"
-           ng-if="!$ctrl.isEnvFromReadonly() && $ctrl.hasOptions()">
+    <div
+      class="environment-from-editor-header prefix-header"
+      ng-if="!$ctrl.isEnvFromReadonly() && $ctrl.hasOptions()">
         Prefix
-        <small class="pficon pficon-help"
+        <small class="pficon pficon-help tooltip-default-icon"
```

Comment:
We should add a style to switch the cursor to the help icon on hover like other tooltips.